assignments/callbacks.js

```diff
@@ -1,46 +1,35 @@
-// Create a callback function and invoke the function to test your work. You have been provided an example of a problem and a solution to see how this works with our items array.  Study both the problem and the solution to figure out the rest of the problems.
-
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
-/* 
-
-  //Given this problem: 
-  
-  function firstItem(arr, cb) {
-    // firstItem passes the first item of the given array to the callback function.
-  }
-
-  // Potential Solution:
-  function firstItem(arr, cb) {
-    return cb(arr[0]);
-  }
-
-  firstItem(items, function(first) {
-    console.log(first)
-  });
-
-*/
-
+function firstItem(arr, cb) {
+  cb = arr[0];
+  return cb;
+}
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  cb = items.length;
+  return cb;
 }
 
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  cb = items[items.length -1];
+  return cb;
 }
 
 function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+  cb = x + y;
+  return cb;
 }
 
 function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
+  cb = x * y;
+  return cb;
 }
 
 function contains(item, list, cb) {
-  // contains checks if an item is present inside of the given array/list.
-  // Pass true to the callback if it is, otherwise pass false.
+  
+  cb = items.includes(item);
+
+  return cb;
```

Comment:
compare these to the solution codes. We're passing the results into the cb, but not setting the cb = to the results of the function